### ISSUE TITLE
Insert page doesn't report MySQL error

### ISSUE THREAD

**Issue by kamil-tekiela** [phpmyadmin/phpmyadmin#19344]
### Describe the bug

A blank page is displayed instead of the usual red box with MySQL error. 

### To Reproduce

Use this table:

```
CREATE TABLE IF NOT EXISTS `bin` (
  `bi` varbinary(2323) NOT NULL,
  `ch` varchar(23) COLLATE utf8mb4_unicode_ci NOT NULL,
  `tt` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `tb` mediumblob NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

1. Go to Insert page
2. Click on 'Go' without filling in any details
4. See blank page

### Expected behavior

A red box with the error message. 

### Server configuration

- Operating system:
- Web server:
- Database version:
- PHP version:
- phpMyAdmin version: 6.0 but didn't check earlier version.

**Comment by liviuconcioiu** [comment]
I get the same issue (blank page), when I import a large CSV file.

**Comment by kamil-tekiela** [comment]
Fixed by #19372
